deployment/secrets/encrypted/deploy_creds_env.sec

```diff
@@ -0,0 +1,2 @@
+codeship:v2
+ZUunEDGEaP6Zb1rIPa2D0JhDaXo2NELI0sWEN8UDvEFCCiIMnqTszv+oPVQHGEASjh1PXPDYcK68qr42mhtR1fv3cOThupntbTitHTokIex+YUmqiasyTzSReQjcveEw5BG4r+8HV1SJh9puZbm/ZLFRrooRJrQJXYTZcP2evyNSS5pbXq83x0ulCr1s4qqB
```

Comment:
I took a look at the decrypted version, you'll need to add aws keys so it should look like this eventually
```
GITHUB_TOKEN=
AWS_ACCESS_KEY_ID=
AWS_SECRET_ACCESS_KEY=
AWS_REGION=us-east-1
```